package.json

```diff
@@ -46,9 +46,11 @@
     "babel-jest": "^29.7.0",
     "concurrently": "^8.2.2",
     "eslint": "^9.7.0",
+    "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-markdown": "^5.1.0",
-    "eslint-plugin-prettier": "^5.1.3",
+    "eslint-plugin-prettier": "^5.2.1",
     "eslint-plugin-react": "^7.34.2",
+    "eslint-plugin-react-hooks": "^4.6.2",
```

Comment:
Had to move some dependencies to the root, as it is used by the linting configuration of the project. It was previously included incidentally as a dependency of the ESLint plugin.